dub.sdl

```diff
@@ -71,7 +71,7 @@ subPackage {
   name "frontend"
   targetType "library"
   sourcePaths "src/dmd"
-  stringImportPaths "res"
+  stringImportPaths "src/dmd/res"
```

Comment:
If you want to merge dlang.org PR without duplication, you need to temporarily add both directories here as dlang.org uses dmd as a library:

```
stringImportPaths "res"
stringImportPaths "src/dmd/res"
```